api/auth.go

```diff
@@ -461,6 +461,85 @@ func teamList(w http.ResponseWriter, r *http.Request, t auth.Token) error {
 	return json.NewEncoder(w).Encode(result)
 }
 
+// title: team info
+// path: /teams/{name}
+// method: GET
+// produce: application/json
+// responses:
+//   200: Info team
+//   404: Not found
+//   401: Unauthorized
+func teamInfo(w http.ResponseWriter, r *http.Request, t auth.Token) error {
+	teamName := r.URL.Query().Get(":name")
+	team, err := auth.GetTeam(teamName)
+	if err != nil {
+		return err
+	}
+
+	_, err = t.Permissions()
+	if err != nil {
+		return err
+	}
+
+	apps, err := app.List(&app.Filter{})
+	if err != nil {
+		return err
+	}
+	includedApps := make([]app.App, 0)
+	for _, app := range apps {
+		for _, appTeam := range app.Teams {
+			if appTeam == team.Name {
+				includedApps = append(includedApps, app)
+				break
+			}
+		}
+	}
+
+	pools, err := provision.ListAllPools()
+	if err != nil {
+		return err
+	}
+	includedPools := make([]provision.Pool, 0)
+	for _, pool := range pools {
+		poolTeams, _ := pool.GetTeams()
+		for _, poolTeam := range poolTeams {
+			if poolTeam == team.Name {
+				includedPools = append(includedPools, pool)
+				break
+			}
+		}
+	}
+
+	users, err := auth.ListUsers()
+	if err != nil {
+		return err
+	}
+	includedUsers := make([]*apiUser, 0)
+	for _, user := range users {
+		for _, role := range user.Roles {
+			if role.ContextValue == team.Name {
```

Comment:
I don't think that's how we should get users for a team. The meaning here should be users with roles with ContextType == CtxTeam and a ContextValue equal to the current team. See my suggestion below for the output grouped by role.
I think this loop would look something like: (untested pseudocode)
```
roleUserMap := map[string][]string{}
for _, user := range users {
  for _, roleInstance := range user.Roles {
    role := permission.FindRole(roleInstance.Name)
    if role.ContextType == permission.CtxTeam && roleInstance.ContextValue == team.Name {
      roleUserMap[role.Name] = append(roleUserMap[role.Name], user.Email)
    }
  }
}
```
You could cache FindRole results in a map to avoid repeated lookups to the same role in different users.